OneSchedule.Domain/StateMachine/GetState/GetNotificationsState.cs

```diff
@@ -0,0 +1,27 @@
+﻿using OneSchedule.Attributes;
+using OneSchedule.Domain.Abstractions.StateMachine;
+using OneSchedule.Domain.Models;
+using OneSchedule.Entities;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OneSchedule.Domain.StateMachine.GetState
+{
+    [StateName("GetNotifications")]
+    public class GetNotificationsState : IState
+    {
+        private const string NextState = "AskEventMenu";
+
+        public Task HandleAsync(IStateContext stateContext, DtoDomain dtoDomain)
+        {
+            var dates = dtoDomain.MessageText.Split(" ");
+            var notifications = dates.Select(d=>
```

Comment:
Format code. If you have sequence of calls it is better to put each method call on new line